test/dotformat.jl

```diff
@@ -3,35 +3,35 @@ using Test
 using Graphs
 
 property_set=[
-    (src="thirty",dst="inf",color="missing",penwidth="missing",style="missing",),
-    (src="twenty",dst="mort",color="missing",penwidth="missing",style="missing",),
-    (src="rate",dst="personper",color="missing",penwidth="missing",style="missing",),
-    (src="person",dst="unit",color="missing",penwidth="missing",style="missing",),
-    (src="personper",dst="unit",color="missing",penwidth="missing",style="missing",),
-    (src="second",dst="unit",color="missing",penwidth="missing",style="missing",),
-    (src="dollars",dst="unit",color="missing",penwidth="missing",style="missing",),
-    (src="dGDP",dst="prices",color="missing",penwidth="missing",style="missing",),
-    (src="prices",dst="regres",color="missing",penwidth="missing",style="missing",),
-    (src="fed",dst="dGDP",color="missing",penwidth="missing",style="missing",),
-    (src="ode",dst="epi",color="missing",penwidth="missing",style="missing",),
-    (src="dollars",dst="cost",color="missing",penwidth="missing",style="missing",),
-    (src="mort",dst="rate",color="black",penwidth="2.0",style="dashed",),
-    (src="inf",dst="rate",color="black",penwidth="2.0",style="dashed",),
-    (src="birth",dst="rate",color="black",penwidth="2.0",style="dashed",),
-    (src="mort",dst="ind",color="orange",penwidth="4.0",style="solid",),
-    (src="ind",dst="epi",color="orange",penwidth="4.0",style="solid",),
-    (src="ind",dst="epi",color="orange",penwidth="4.0",style="solid",),
-    (src="inf",dst="ind",color="orange",penwidth="4.0",style="solid",),
-    (src="birth",dst="ind",color="orange",penwidth="4.0",style="solid",),
-    (src="temp",dst="inf",color="orange",penwidth="4.0",style="solid",),
-    (src="age",dst="mort",color="orange",penwidth="4.0",style="solid",),
-    (src="demo",dst="birth",color="orange",penwidth="4.0",style="solid",),
-    (src="epi",dst="cases",color="orange",penwidth="4.0",style="solid",),
-    (src="cases",dst="regres",color="orange",penwidth="4.0",style="solid",),
-    (src="weather",dst="temp",color="orange",penwidth="4.0",style="solid",),
-    (src="demo",dst="age",color="orange",penwidth="4.0",style="solid",),
-    (src="regres",dst="cost",color="orange",penwidth="4.0",style="solid",),
-    (src="html",dst="fed",color="black",penwidth="2.0",style="solid",)
+    (src="thirty",dst="inf",color="missing",penwidth="missing",style="missing",label=""),
```

Comment:
Why does a new property show up?

Reply:
To test (new) code that handles labels in addition to the old tests. Label handling was already present in the library but wasn't tested.